example.config.yml

```diff
@@ -69,6 +69,19 @@ apache_vhosts:
   - {servername: "xhprof.drupalvm.dev", documentroot: "/usr/share/php/xhprof_html"}
   - {servername: "pimpmylog.drupalvm.dev", documentroot: "/usr/share/php/pimpmylog"}
 
+# Enable SSL Support on the Drupal site vhosts.
+# See https://github.com/geerlingguy/ansible-role-apache for additional options.
+# Update the file paths below if you are using a custom SSL certificate.
+apache_vhosts_ssl:
+  - {
+    servername: "{{ drupal_domain }}",
+#    serveralias: "www.{{ drupal_domain }}",
+    documentroot: "{{ drupal_core_path }}",
+    certificate_file: "/etc/ssl/certs/ssl-cert-snakeoil.pem",
```

Comment:
@shawnachieve @aweingarten - I like the idea of auto-generating a self-signed cert for {{ drupal_domain }} (at a minimum), but I fear it'll start getting a little more difficult to get it working well across platforms and in a way that the cert is valid everywhere (at least in a short timeframe).

I think, as a first step, I want to add some really good/focused docs to the docs site, then we can work on a more comprehensive solution.